sdks/python/tox.ini

```diff
@@ -210,6 +210,7 @@ passenv = REQUESTS_CA_BUNDLE
 allowlist_externals = 
   wget
   az
+  bash
```

Comment:
why do we need bash here?

Reply:
Okay. looks like `commands_post` use bash and the `allowlist_externals` in the current context overwrites the `allowlist_externals` in the global context of tox.ini